### PR TITLE
minor phpcs changes

### PR DESCRIPTION
**Pull request by TomHAnderson**

### CODE REVIEW

File: src/DoctrineORMModule/Listener/MigrationConfigurationListener.php [hunk 1]
```diff
@@ -44,22 +44,29 @@ public function __construct(ServiceLocatorInterface $container)
...
-        $objectManagerName = 'doctrine.entitymanager.orm_default';
         if ($input->hasParameterOption(['--object-manager'])) {
-            $objectManagerName = $input->getParameterOption(['--object-manager']);
+            $objectManagerAlias = $input->getParameterOption(['--object-manager']);
         }
-        $migrationConfiguration = $this->container->get(
-            str_replace('entitymanager', 'migrations_configuration', $objectManagerName)
-        );
+
+        # FIXME:  Add comment about why we make this alias change
```
Comment (TomHAnderson) [inline src/DoctrineORMModule/Listener/MigrationConfigurationListener.php]: Can you edit and add some comment about this?  Are we fetching `doctrine.migrations_configuration.orm_default` ?

**Comment by TomHAnderson** [comment]
Mostly I made three kinds of changes

* Right pad bang operator `if (! $var)`
* Sort use definitions by most important with this module listed last
* Added a few extra blank lines.

**Comment by tdutrion** [comment]
I'm not sure about `Sort use definitions by most important with this module listed last`... How would you define which one is more important?
Most IDEs will sort use statements in alphabetical order as well.

File: src/DoctrineORMModule/Listener/MigrationConfigurationListener.php (hunk 1 above)
Comment (tdutrion) [reply src/DoctrineORMModule/Listener/MigrationConfigurationListener.php]: We are not. First we define the default object manager alias (`doctrine.entitymanager.orm_default`), then try to see whether another object manager alias has been passed as a command line argument, and finally change the middle part to retrieve the migration configuration alias.

**Comment by TomHAnderson** [comment]
I've tried finding a best method of sorting USE statements for years.  My first try I call "The Devil's Backbone" and it was soundly rejected by my peers.  That looks like this

```php
use Class
    , Class2
    , Class3
    ;
```

I don't use an IDE so I haven't seen the 'order by alphabetical name' and noticed it.  I can see how this would be popular with IDEs because they are used by more junior developers.

Next I tried organizing by length with the longest on top

```php
use Class\One\Here;
use Class\Two;
use Exception;
use Three;
```

but I found that was too unorganized.  So the method I'm using now groups by most important & closest to PHP native.

```php
use stdClass; # special case always use lower case for `std`
use Exception;
use Zend\Class; # Zend is the framework and has second priority to PHP classes
use Doctrine\Class; # Doctrine is a library used in the application and is not a framework so it goes here
use Slm\Queue;
use ZF\OAuth2\Doctrine\Entity as OAuth2Entity; # here the pattern is flexible and up to the prerogative of the developer; is Slm more important in this than OAuth2?
use CurrentModule\Class; # Current module always comes last
```

Benefits of the current module coming last is you can read the USE from bottom to top to find what's probably more important to your programming.  The class definition makes a nice divider to read down vs read up.  

This method also groups all classes by module name/namespaces.

**Pull request merged**